### ISSUE TITLE
Document custom recognition weights for Grammar/Rule/Element

### ISSUE THREAD

**Issue by kendonB** [dictation-toolbox/Caster#808]
This page:

https://caster.readthedocs.io/en/latest/readthedocs/Getting_Started/Choose_Recognition_Engine/

mentions "Custom recognition weights for Grammar/Rule/Element" for kaldi but I don't see this documented anywhere. Docs should be added for this.

**Comment by LexiconCode** [comment]
Yes this does need to be documented in some fashion. I do plan to add a couple sections for troubleshooting accuracy and grammars.

The question is how detailed should we get with the caster documentation or simply refer people to https://dragonfly2.readthedocs.io/en/latest/kaldi_engine.html#grammar-rule-element-weights

**Comment by kendonB** [comment]
Just the minimum required for it to work in caster I think. with a reference to the other page.

I think having an example of a minimal caster rule file with weights would be helpful as well.
